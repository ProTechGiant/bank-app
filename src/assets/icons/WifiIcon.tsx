import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function WifiIcon({ width = 24, height = 24, color = "#FF371E" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M1 9.27711L3 11.2771C7.97 6.30711 16.03 6.30711 21 11.2771L23 9.27711C16.93 3.20711 7.08 3.20711 1 9.27711ZM9 17.2771L12 20.2771L15 17.2771C13.35 15.6171 10.66 15.6171 9 17.2771ZM5 13.2771L7 15.2771C9.76 12.5171 14.24 12.5171 17 15.2771L19 13.2771C15.14 9.41711 8.87 9.41711 5 13.2771Z"
        fill={color}
      />
    </Svg>
  );
}
