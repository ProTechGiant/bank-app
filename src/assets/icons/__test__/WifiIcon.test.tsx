import { create } from "react-test-renderer";

import { WifiIcon } from "../WifiIcon";

describe("components/Carousel", () => {
  it("renders a Carousel", () => {
    const result = create(<WifiIcon />);

    expect(result).toMatchSnapshot();
  });
});
