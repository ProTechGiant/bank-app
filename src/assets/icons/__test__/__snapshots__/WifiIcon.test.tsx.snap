// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Carousel renders a Carousel 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={25}
  vbWidth={24}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M1 9.27711L3 11.2771C7.97 6.30711 16.03 6.30711 21 11.2771L23 9.27711C16.93 3.20711 7.08 3.20711 1 9.27711ZM9 17.2771L12 20.2771L15 17.2771C13.35 15.6171 10.66 15.6171 9 17.2771ZM5 13.2771L7 15.2771C9.76 12.5171 14.24 12.5171 17 15.2771L19 13.2771C15.14 9.41711 8.87 9.41711 5 13.2771Z"
      fill={
        {
          "payload": 4294915870,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
